packages/zowe-explorer/src/utils/AuthUtils.ts

```diff
@@ -41,7 +41,7 @@ export class AuthUtils {
                         return;
                     }
                 }
-            } else if (httpErrorCode === imperative.RestConstants.HTTP_STATUS_401) {
+            } else if (httpErrorCode === imperative.RestConstants.HTTP_STATUS_401 || imperativeError.message.includes("authentication")) {
```

Comment:
Thanks for looking into this Likhitha!
I'm wondering if we want to make this check more specific 🤔 this might match against other errors returned from the SSH client, not just the error that represents invalid credentials.

Can we match against more of the error message returned by the SSH client?

Reply:
Addressed it